composer.json

```diff
@@ -86,6 +89,9 @@
         ],
         "post-create-project-cmd": [
             "@php artisan key:generate --ansi"
+        ],
+        "post-install-cmd": [
+            "[ $COMPOSER_DEV_MODE -eq 1 ] || composer require laravel/nova mrmonat/nova-translatable tightenco/nova-releases inspheric/nova-url-field tightenco/suggested-resources-shortcuts"
```

Comment:
Can you tell me what's going on here?

Reply:
I'm using `$COMPOSER_DEV_MODE` variable to check which environment we're building in. If it's local development, all nova packages are skipped and the programmer needs to manually require nova and all the additional packages it uses if they want to work locally with it. All nova packages are automatically installed if it's production. `$COMPOSER_DEV_MODE` is set by the `--no-dev` flag during the `composer install` call.

Reply:
OHHHHH that's super clever. Thanks for explaining!!